src/components/Utils/RepeatableFieldCatchError/RepeatableFieldCatchError.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { get } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import { RepeatableField } from '@folio/stripes-components';
+import css from './RepeatableFieldCatchError.css';
+
+export const RepeatableFieldCatchError = (props) => { // eslint-disable-line import/prefer-default-export
```

Comment:
`RepeatableFieldCatchError.js` - is not the best name. You actually don't catch an error, you've added a block with error message. 
Also please use another dir for common code - it's src/common